### ISSUE TITLE
🚀 v0.7 Release

### ISSUE THREAD

**Issue by JWCook** [reclosedev/requests-cache#312]
Requests-cache 0.7 is now available; see [changelog](https://github.com/reclosedev/requests-cache/blob/master/HISTORY.md#070-2021-07-07) for details. Thank you to to all [contributors](https://github.com/reclosedev/requests-cache/blob/master/CONTRIBUTORS.md) who provided pull requests, feedback, and other contributions for this release!

Automated tests have been significantly improved, but it's still possible that some bugs or unexpected behavior could slip past. A number of new features have also been added, so it's possible those could have untested edge cases that aren't yet handled correctly. If you encounter any problems after upgrading to 0.7, either with existing code or with any of the new features, please report them either here or in a new issue.

**Comment by miles82** [comment]
Hi, I think I found a bug in 0.7.0. It seems that it always respects cache-control headers (regardless of `cache_control` setting):

```
from requests_cache import CachedSession

session = CachedSession(cache_name='cache_control_test')
session.cache.clear()

def is_cached(url):
    print(url)
    for _ in range(5):
        print(session.get(url).from_cache)

# response with max-age=0
is_cached('https://httpbin.org/cache/0')
# response with max-age=1000
is_cached('https://httpbin.org/cache/1000')
```

outputs:

```
https://httpbin.org/cache/0
False
False
False
False
False
https://httpbin.org/cache/1000
False
True
True
True
True
```

I would expect the first URL to also be cached after the first try (like it is in 0.6.4), unless we enable `cache_control` on `CachedSession`.

**Comment by JWCook** [comment]
Thanks for the tip! I'll get a patch out for that shortly.

**Comment by JWCook** [comment]
@miles82 This is now fixed in version `0.7.1`, and some better tests have been added to catch that in the future. Thanks!

**Comment by miles82** [comment]
Thanks, that was fast! I can confirm it works as expected now.
